crmd/notify.c

```diff
@@ -20,120 +20,501 @@
 #include <crm/crm.h>
 #include <crm/msg_xml.h>
 #include "notify.h"
+#include "crmd_messages.h"
 
 char *notify_script = NULL;
 char *notify_target = NULL;
+GListPtr notify_list = NULL;
+
+typedef struct {
+    char *id;
+    char *path;
+    int timeout;
+    char *tstamp_format;
+    char *info;
```

Comment:
I can see the usefulness, but I don't feel it adds enough to be worth the extra complexity (one more variable for a user to figure out). If someone wants to use a generic script with multiple clusters, they can create their own local config file for it to read, which gives them the flexibility to decide how they want to do that (format, config management, etc.).

If we did want to do this in pacemaker, I think I'd rather offer a generic name-value pair functionality, and let the user specify what variables they want. I.e. each <alert> could have an <instance_attributes> section. Recipient could even fit into that, but we'd be back to just one recipient (although the script could do something like allow the recipient to be a comma-separated list). For consistency and future flexibility, timeout could even be in a <meta_attributes> section. It sounds like a good bit of extra work and complexity, but it would be consistent with other XML objects. What do you think?